labs-ios-starter/Model/CoreData MO/Apollo/Web Auth/Member/Member+CoreDataClass.swift

```diff
@@ -0,0 +1,95 @@
+// Copyright © 2020 Shawn James. All rights reserved.
+// Member+CoreDataClass.swift
+
+import CoreData
+import UIKit
+
+/// Standard user, leader if topic originator
+public final class Member: NSManagedObject, Codable {
+    // MARK: - Coding Keys
+
+    enum MemberCodingKeys: String, CodingKey {
+        case id, email
+        case firstName = "firstname"
+        case lastName = "lastname"
+        case avatarURL = "avatarUrl"
+    }
+
+    // MARK: - Initializer
+
+    /// Used to create a 'Member' managed object
+    /// - Parameters:
+    ///   - oktaID: the user's current Okta Bearer Token (only used for login)
+    ///   - id: defaults to new UUID, make sure to assign the correct one when decoding
+    ///   rather than generating a new one
+    ///   - email: optional, used for login and updating email only (defaults to nil)
+    ///   - firstName: optional, used for registration and updating name only (defaults to nil)
+    ///   - lastName: optional, used for registration and updating name only (defaults to nil)
+    ///   - avatarURL: optional, used for displaying the user's profile image
+    @discardableResult convenience init(oktaID: String?,
+                                        id: Int64?,
+                                        email: String?,
+                                        firstName: String?,
+                                        lastName: String?,
+                                        avatarURL: URL?,
+                                        context: NSManagedObjectContext = CoreDataManager.shared.mainContext) {
+        self.init(context: context)
+        self.oktaID = oktaID
```

Comment:
This property can be removed altogether actually. I was mistaken that we needed it to interact with the web backend beyond login. I *just* removed it from my feature/get-topics branch (and moved the `createProfile` method in `ProfileController ` to `Documentation/unused.md` - but don't worry about that part). If you want, you can leave it in and we can clean it up later, or you can remove it now and just remove the createProfile method so the project will compile 😄

Reply:
I'll remove it now and just remove the createProfile method so the project will compile